<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mymsgdialog</class>
 <widget class="QDialog" name="mymsgdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="allMsgBtn">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>全部消息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="unreadMsgBtn">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>未读消息</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="msgTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>400</width>
     <height>270</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>270</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>270</height>
    </size>
   </property>
   <property name="autoScroll">
    <bool>true</bool>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <attribute name="verticalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>标题</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>内容</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>发送时间</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>状态</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="markReadBtn">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>270</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>标记已读</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>270</y>
     <width>56</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>删除消息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
